### PR TITLE
Updating announcement for OWASP 5 rule changes and MR 1 rule toggle

### PR DESCRIPTION
**Pull request by unknownhad**

### CODE REVIEW

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -24,31 +24,48 @@ rss: file
...
+      <td>Fixing OWASP Rule ...5ab75703 : 944120: Remote Command Execution: Java serialization (CVE-2015-4852)</td>
+    </tr>
+    <tr>
+      <td>2023-07-05</td>
+      <td>2023-07-10</td>
+      <td>OWASP Rules</td>
+      <td>N/A</td>
+      <td>...5ab75703</td>
+      <td>944210: Magic bytes Detected Base64 Encoded, probable java serialization in use</td>
+      <td>Fixing OWASP Rule ...73cd4e53 : 944210: Magic bytes Detected Base64 Encoded, probable java serialization in use</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>944210: Magic bytes Detected Base64 Encoded, probable Java serialization in use</td>
      <td>Fixing OWASP Rule ...73cd4e53 : 944210: Magic bytes Detected Base64 Encoded, probable Java serialization in use</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -24,31 +24,48 @@ rss: file
...
+      <tr>
+      <td>2023-07-05</td>
+      <td>2023-07-10</td>
+      <td>OWASP Rules</td>
+      <td>N/A</td>
+      <td>...e068f5d3</td>
+      <td>944210: Magic bytes Detected Base64 Encoded, probable java serialization in use</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]: This is the same description as above... can you check?

File: content/waf/change-log/scheduled-changes.md [hunk 3]
```diff
@@ -24,31 +24,48 @@ rss: file
...
+    </tr>
+    <tr>
+      <td>2023-07-05</td>
+      <td>2023-07-10</td>
+      <td>OWASP Rules</td>
+      <td>N/A</td>
+      <td>...5ab75703</td>
+      <td>944210: Magic bytes Detected Base64 Encoded, probable java serialization in use</td>
+      <td>Fixing OWASP Rule ...73cd4e53 : 944210: Magic bytes Detected Base64 Encoded, probable java serialization in use</td>
+      <tr>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
    </tr>
    <tr>
```

File: content/waf/change-log/scheduled-changes.md [hunk 4]
```diff
@@ -24,31 +24,49 @@ rss: file
...
+      <td>944210: Magic bytes Detected Base64 Encoded, probable Java serialization in use</td>
+      <td>Fixing OWASP Rule ...73cd4e53 : 944210: Magic bytes Detected Base64 Encoded, probable Java serialization in use</td>
+    </tr>
+    <tr>
+      <td>2023-07-05</td>
+      <td>2023-07-10</td>
+      <td>OWASP Rules</td>
+      <td>N/A</td>
+      <td>...e068f5d3</td>
+      <td>944210: Magic bytes Detected Base64 Encoded, probable java serialization in use</td>
```
Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>944300: Base64 encoded string matched suspicious keyword</td>
```

**Review by helloandre — APPROVED** [review]

**Pull request merged**